### DISCUSSION TITLE
Internet access icon connection problems

### DISCUSSION THREAD

**Discussion by grstatdoc-gr** [valinet/ExplorerPatcher#2239] [General]
Hi! I installed ExplorerPatcher on my Windows 11 machine (Win 11 Pro 22H2, Lenovo IdeaPad 15), and am overall liking it very much!

While the speakers icon is working OK (I can left-click on it and it brings up the speakers control, I'm having no such success with the  internet access icon (e.g. right-clicking does **not** bring up the Win10 internet connection dialog. 
<img width="27" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/71800985/f4b62d06-a65f-4c97-b7b7-2cf4f7f96cab">
So, if I wish to disconnect to the internet to play a certain game, I must right-click the icon and select Open Network and Internet Settings
<img width="190" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/71800985/c94d1747-7f07-4df0-b698-f2c61b7b4f0b"> to disconnect
When I wish to reconnect, I must go back into the Open Network and Internet settings to reconnect.

**Comment by grstatdoc-gr** [comment]
Alternately, I can go to the Control Center to disconnect or connect, but if that's a step by left-clicking on that icon, I think it would be helpful and more in the spirit of the Win 10 taskbar

**Comment by pyrates999** [comment]
I don't think windows has ever supported disconnecting wired connections the way you can disconnect wireless connections.  You can only do that by disabling the network port.
